src/vm/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR})
 include_directories(${ARCH_SOURCES_DIR})
 
+# EventPipe stress testing support
+# add_definitions(-DFEATURE_AUTO_TRACE)
```

Comment:
Was this left here on purpose?

Reply:
If you were trying to set this FEATURE_AUTO_TRACE to 0 on default, try putting it in clrfeatures.cmake instead and set it to 0 as default?